src/client/src/components/pages/workers/addWorkers/index.jsx

```diff
@@ -24,17 +26,53 @@ export default class index extends Component {
     this.setState(fields);
   };
 
+  addWorker = async obj => {
+    const confirm = await swal({
+      type: "warning",
+      html: "Are you sure that you want to add this worker ?",
+      showCancelButton: true,
+      focusConfirm: false,
+      confirmButtonText: '<i className="fa fa-thumbs-up"></i> Yes',
```

Comment:
@mohannadhanafi  Here is a same about `className` for `confirmButtonText`
_you can replace it with `class` to show the icons in your views_

Reply:
Done :+1: